lutris/gui/widgets/default_path.py

```diff
@@ -0,0 +1,129 @@
+import os
+from enum import Enum, auto
+from gettext import gettext as _
+
+from lutris.util.log import logger
+
+
+class PATH_TYPE(Enum):
+    """Types of paths, UNKNOWN should be used by default"""
+
+    UNKNOWN = 0
+    BANNER = auto()
+    ICON = auto()
+    CACHE = auto()
+    INSTALLER = auto()
+    INSTALL_TO = auto()
+
+
+class default_path_handler:
+    """Handles finding the correct path to set in the file chooser dialog"""
+
+    last_selected_path = {PATH_TYPE.UNKNOWN: None}
+
+    @staticmethod
+    def is_valid(path):
+        """`is_valid` Checks if the path is valid
+
+        Args:
+        - `path` (`str`): Path to check, can be `None`
```

Comment:
What type of docstring style is this following? Is it any of the following mentioned here: https://stackoverflow.com/a/24385103 or is it some custom flavour?

Reply:
Based on the Google style with slight modification to handle markdown display properly.  Looks like the attached picture when hovering over the init function for example.
![comment_tooltip](https://user-images.githubusercontent.com/48720687/95073505-d6f88c80-06c1-11eb-945b-16b5ee324993.png)